packages/ripple-address-codec/src/index.ts

```diff
@@ -43,8 +43,10 @@ function encodeXAddress(
   if (tag > MAX_32_BIT_UNSIGNED_INT) {
     throw new Error('Invalid tag')
   }
-  const theTag = tag === false ? 0 : tag
-  const flag = tag === false ? 0 : 1
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- Passing null is a common js mistake
+  const theTag = tag === false || tag == null ? 0 : tag
```

Comment:
Should undefined be thrown in for good measure?

Reply:
I think doing `== null` instead of `=== null` does the undefined check as well.